graphile-build/graphile-build-pg/src/plugins/PgAttributesPlugin.ts

```diff
@@ -42,7 +41,7 @@ declare global {
       _attributeName(
         this: GraphileBuild.Inflection,
         details: {
-          codec: PgCodecWithAttributes;
+          codec: DefaultPgCodec;
```

Comment:
Should this (and the related changes below) have been `DefaultPgCodecWithAttributes` or similar?

Reply:
Yes, but this gave me a lot of errors, so I opted to widen the constraint for now, as I felt it wasn't necessarily a requirement for the purpose of this PR.